model/DatabaseAdmin.php

```diff
@@ -228,7 +228,11 @@ public function doBuild($quiet = false, $populate = true, $testMode = false) {
 		$dbSchema->schemaUpdate(function() use($dataClasses, $testMode, $quiet){
 			foreach($dataClasses as $dataClass) {
 				// Check if class exists before trying to instantiate - this sidesteps any manifest weirdness
-				if(!class_exists($dataClass)) continue;
+				if(!class_exists($dataClass) 
+					|| (($reflection = new ReflectionClass($dataClass)) 
+					&& $reflection->isAbstract())) {
```

Comment:
This actually breaks dev/build in another way; Any `$db` on this abstract class will not be built, although it will be silently ignored.